xwayland/xwm.c

```diff
@@ -632,6 +633,22 @@ static void xwm_handle_configure_notify(struct wlr_xwm *xwm,
 	xsurface->y = ev->y;
```

Comment:
More a nitpick than anything, this function doesn't have a log message unlike handle_configure_request. I think it could make sense to add one if we find xsurface that logs new x/y/width/height at debug level.
Not overly fussy about this